### PR TITLE
NTP, MTU and PKG tests

### PR DESCRIPTION
**Pull request by vkhalin**
Plz review my draft version of tests and if it is possible - merge them. They were tested on jse2 and gives correct results. Comment on what needs to be changed.

### CODE REVIEW

File: mk_verificator/tests/system/test_ntp_sync.py [hunk 1]
```diff
@@ -1,88 +1,43 @@
-import salt.client as client
-import texttable as tt
+def test_ntp_sync(local_salt_client):
+
+    data = {}
+    node_times_list = []
+    h = 0
```
Comment (vkhalin) [inline mk_verificator/tests/system/test_ntp_sync.py]: I will do that in next pull request as it is right now already very big. Can we merge that without those changes?

File: mk_verificator/tests/networking/mtu/gauges.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
+{
```
Comment (vkhalin) [inline mk_verificator/tests/networking/mtu/gauges.json]: I need help on that and I will do that in next pull request as it is right now already very big. Can we merge that without those changes?

File: mk_verificator/tests/system/test_ntp_sync.py (hunk 1 above)
Comment (msenin94) [inline mk_verificator/tests/system/test_ntp_sync.py]: that was why I asked you to split all changes by different pull request
